test/services/GetPenaltyDetailsServiceSpec.scala

```diff
@@ -199,5 +202,108 @@ class GetPenaltyDetailsServiceSpec extends SpecBase with LogCapturing {
       val result: Exception = intercept[Exception](await(service.getDataFromPenaltyServiceForVATCVRN("123456789")))
       result.getMessage shouldBe "Something has gone wrong."
     }
+
+    class SetUpWithMockConfig {
+      val mockConfig: Configuration = mock(classOf[Configuration])
+      val mockServiceConfig: ServicesConfig = mock(classOf[ServicesConfig])
+
+      val appConfig: AppConfig = new AppConfig(mockConfig, mockServiceConfig)
+      val service = new GetPenaltyDetailsService(mockGetPenaltyDetailsConnector)(ec = ec, appConfig = appConfig)
+      reset(mockGetPenaltyDetailsConnector)
+    }
+
+    val lpp1PrincipleChargeDueToday: LPPDetails = LPPDetails(
+      penaltyCategory = LPPPenaltyCategoryEnum.FirstPenalty,
+      principalChargeReference = "1234567890",
+      penaltyChargeReference = Some("123456789"),
+      penaltyChargeCreationDate = Some(LocalDate.of(2022, 10, 30)),
+      penaltyStatus = LPPPenaltyStatusEnum.Accruing,
+      appealInformation = Some(Seq(AppealInformationType(appealStatus = Some(AppealStatusEnum.Unappealable), appealLevel = Some(AppealLevelEnum.HMRC)))),
+      principalChargeBillingFrom = LocalDate.of(2022, 10, 30),
+      principalChargeBillingTo = LocalDate.of(2022, 10, 30),
+      principalChargeDueDate = LocalDate.now(),
+      communicationsDate = Some(LocalDate.of(2022, 10, 30)),
+      penaltyAmountOutstanding = None,
+      penaltyAmountPaid = None,
+      penaltyAmountPosted = None,
+      LPP1LRDays = Some("15"),
+      LPP1HRDays = Some("31"),
+      LPP2Days = Some("31"),
```

Comment:
I know we don't usually model the data like-for-like in unit tests but just thinking if this should not have LPP2 stuff?

Reply:
What do you think to this?

Reply:
Created separate LPP1 and LPP2 models to be more realistic